autofs-5.1.2 - Fix compiler warning in try_remount()

From: Tomohiro Kusumi <kusumi.tomohiro@gmail.com>

Fix below compiler (gcc5) warning which isn't autoconf result dependent.
mounts.c:1779:7: warning: variable 'ret' set but not used [-Wunused-but-set-variable]

Signed-off-by: Tomohiro Kusumi <kusumi.tomohiro@gmail.com>
Signed-off-by: Ian Kent <raven@themaw.net>
---
 CHANGELOG    |    1 +
 lib/mounts.c |    5 ++---
 2 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/CHANGELOG b/CHANGELOG
index 2e51f62..288d175 100644
--- a/CHANGELOG
+++ b/CHANGELOG
@@ -2,6 +2,7 @@ xx/xx/2016 autofs-5.1.3
 =======================
 - fix release date in CHANGELOG.
 - build: check for clock_gettime in librt.
+- fix compiler warning in try_remount().
 
 15/06/2016 autofs-5.1.2
 =======================
diff --git a/lib/mounts.c b/lib/mounts.c
index 1d1b4da..c67548e 100644
--- a/lib/mounts.c
+++ b/lib/mounts.c
@@ -1776,16 +1776,15 @@ static int remount_active_mount(struct autofs_point *ap,
 			*ioctlfd = -1;
 		}
 	} else {
-		int ret;
 		/*
 		 * What can I do if we can't remount the existing
 		 * mount(s) (possibly a partial failure), everything
 		 * following will be broken?
 		 */
 		if (type == t_indirect)
-			ret = do_remount_indirect(ap, fd, path);
+			do_remount_indirect(ap, fd, path);
 		else
-			ret = do_remount_direct(ap, fd, path);
+			do_remount_direct(ap, fd, path);
 	}
 
 	debug(ap->logopt, "re-connected to mount %s", path);
